packages/libs/react-ui/src/components/NavHeader/index.ts

```diff
@@ -0,0 +1,7 @@
+export type { INavHeaderProps } from './NavHeader';
+export type { INavHeaderNavigationProps } from './NavHeaderNavigation';
+export type { INavHeaderContentProps } from './NavHeaderContent';
+
+export { NavHeader } from './NavHeader';
+export { NavHeaderNavigation } from './NavHeaderNavigation';
+export { NavHeaderContent } from './NavHeaderContent';
```

Comment:
This needs to be updated to use our subcomponent naming convention
`NavHeader.Root`, `NavHeader.NavLinks`, `NavHeader.Content`